pkg/registry/service/ipallocator/allocator_test.go

```diff
@@ -142,27 +143,13 @@ func TestAllocateSmall(t *testing.T) {
 		}
 	}
 
-	if r.count != 2 && r.Free() != 0 && r.max != 2 {
+	if r.Free() != 0 {
```

Comment:
Tim originally had me validate max to prove that the max was right - possibly need to reintroduce that test somewhere else.